lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    def initialize
+      @government_repo = BulkData::GovernmentRepository.new
+    end
+
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless @government_repo.cache_populated?
+      return GovukHealthcheck::WARNING if @government_repo.cache_age > 6.hours
+
+      GovukHealthcheck::OK
+    end
+
+    def details
+      {
+        critical: "No government data availible",
+        warning: warning_details_content,
+      }
+    end
+
+    def enabled?
+      true
+    end
+
+  private
+
+    def warning_details_content
+      data_age = @government_repo.cache_age.to_i
+      "Government data not refreshed in #{data_age / 3600} hours #{data_age % 60} minutes"
```

Comment:
The arithmetic is still wrong here.

For example if you have 4000 seconds it is 1 hours 6 minutes. Whereas this will say "Government data not refreshed in 1 hours 40 minutes" - which also reveals plural issues.

Reply:
Changed the working of this after a face to face conversation. Decided that it was accurate enough to just have hours given that this won't show up unless the data has been missing for more than 6.